ament_package/template/prefix_level/_local_setup_util.py

```diff
@@ -48,10 +48,8 @@ def main(argv=sys.argv[1:]):  # noqa: D103
         FORMAT_STR_USE_ENV_VAR = '${name}'
         FORMAT_STR_INVOKE_SCRIPT = 'AMENT_CURRENT_PREFIX="{prefix}" ' \
             '_ament_prefix_sh_source_script "{script_path}"'
-        FORMAT_STR_REMOVE_LEADING_SEPARATOR = 'if [ "$(echo -n ${name} | ' \
-            'head -c 1)" = ":" ]; then export {name}=${{{name}#?}} ; fi'
-        FORMAT_STR_REMOVE_TRAILING_SEPARATOR = 'if [ "$(echo -n ${name} | ' \
-            'tail -c 1)" = ":" ]; then export {name}=${{{name}%?}} ; fi'
+        FORMAT_STR_REMOVE_LEADING_SEPARATOR = 'export {name}=${{{name}#:}}'
+        FORMAT_STR_REMOVE_TRAILING_SEPARATOR = 'export {name}=${{{name}%:}}'
```

Comment:
@cottsay asked me to eyeball this for POSIX / standard `sh` compliance. These parameter expansion forms have been in the standard since at least 2008 (the oldest version I could find hosted from [posix.opengroup.org](https://pubs.opengroup.org/onlinepubs/9699919799.2008edition/) and I tested both Debian Almquist shell and ksh 2020, two of my favorite shells for vetting POSIX compliance / preventing the unintended spread of bash-isms.